src/Modules/CalcActiveSkill.lua

```diff
@@ -632,9 +632,14 @@ function calcs.buildActiveSkillModList(env, activeSkill)
 			minion.level = m_min(m_max(minion.level,1),100) 
 			minion.itemList = { }
 			minion.uses = activeGrantedEffect.minionUses
-			minion.lifeTable = isSpectre and env.data.monsterLifeTable or env.data.monsterAllyLifeTable
-			local attackTime = minion.minionData.attackTime * (1 - (minion.minionData.damageFixup or 0))
-			local damage = env.data.monsterDamageTable[minion.level] * minion.minionData.damage * attackTime
+			minion.lifeTable = isSpectre and ((minion.minionData.lifeScaling == "AltLife1" and env.data.monsterLifeTable2) or (minion.minionData.lifeScaling == "AltLife2" and env.data.monsterLifeTable3) or env.data.monsterLifeTable) or env.data.monsterAllyLifeTable
+			-- the damage fixup stat applies x% less base Attack Damage and x% more base Attack Speed as confirmed by Openarl Jan 4th 2024
```

Comment:
For minions that don't have `baseDamageIgnoresAttackSpeed`, a single application of the fixup would accomplish both of these, did you confirm whether it's supposed to double dip for those minions by applying twice?